docs/manual/it/introduction/Installation.html

```diff
@@ -117,10 +117,10 @@ <h2>Addons</h2>
 
 		&lt;script type="importmap">
 			{
-			"imports": {
-				"three": "https://unpkg.com/three@&lt;version&gt;/build/three.module.js",
-				"three/addons/": "https://unpkg.com/three@&lt;version&gt;/examples/jsm/"
-			}
+				"imports": {
+					"three": "https://unpkg.com/three@&lt;version&gt;/build/three.module.js",
+					"three/addons/": "https://unpkg.com/three@&lt;version&gt;/examples/jsm/"
+				}
```

Comment:
I don't think these alignments are right. They are now inconsistent regarding the rest of the file.

As long as the indentation looks good in the document view, I don't think we should invest time formatting.